lib/src/curve_widget.dart

```diff
@@ -313,6 +340,10 @@ class _RenderAnimatedTo extends RenderProxyBox {
 
   @override
   void dispose() {
+    if (_scrollController != null) {
```

Comment:
I guess `conroller.dispose()` will clear all the listeners inside and solve the initial errors. If my understanding is correct, maybe the code changes can be made much smaller?

Reply:
Does this mean calling '_scrollController.dispose()' here?
Since the scrollController itself was passed in, I think calling dispose is unexpected behavior. What do you think?

It would be helpful if you could let me know what changes you expect to make.

Reply:
@nappannda aaa, you're right. It shouldn't be disposed inside the package. Then, your code looks the best. Let me check again.

Reply:
@nappannda Now, I've checked the code again, and I agree yours is relevant! So can you solve the conflict as `controller` is now divided into `horizontalController` and `verticalController`? お手数おかけしますがぜひ！

Reply:
I have created a new PR here so this one will be closed.
https://github.com/chooyan-eng/animated_to/pull/12